SwiftMonkeyPaws/MonkeyPaws.swift

```diff
@@ -112,14 +124,16 @@ public class MonkeyPaws: NSObject, CALayerDelegate {
                 gesture.extend(to: point)
             }
         } else {
-            if gestures.count > maxGesturesShown {
+            if gestures.count > self.configuration.paws.maxShown {
                 gestures.removeFirst()
             }
 
-            gestures.append((hash: touchHash, gesture: Gesture(from: point, inLayer: layer)))
+            gestures.append((hash: touchHash, gesture: Gesture(from: point, inLayer: layer, configuration: self.configuration, bezierPathDrawer: self.bezierPathDrawer)))
```

Comment:
`self` not needed in `self.configuration` and `self.bezierPathDrawer`.

If we had `SwiftFormat` integrated this would be automatically fixed ;)